fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -355,7 +355,7 @@ class DDTeamCollectionImpl {
 		for (; idx < self->badTeams.size(); idx++) {
 			servers.clear();
 			for (const auto& server : self->badTeams[idx]->getServers()) {
-				if (server->inDesiredDC && !self->server_status.get(server->id).isUnhealthy()) {
+				if (server->inDesiredDC && !self->server_status.get(server->getId()).isUnhealthy()) {
```

Comment:
[General comment - Not something to be addressed in this PR.]
getXYZ() seems to be conventional in this codebase. Personally, I feel like 'get' is often redundant. `server->Id()` reads more naturally than `server->getId()` (still a method; i'm not suggesting we access the member variable). Depending on how others feel about it, the ongoing refactor might be an opportunity to visit some of these instances. Of course, there is the matter if not breaking from the codebase convention, which is important when there are many contributors.

Reply:
I don't think we have a completely consistent pattern for using `<field>()` or `get<field>()` currently. This is probably a good thing to discuss in the Monday meeting. I agree for short field names like `id`, I probably should have just dropped the `get` prefix.